plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -845,4 +841,29 @@ private static long maxPrecision(int numBytes)
     {
         return Math.round(Math.floor(Math.log10(Math.pow(2, 8 * numBytes - 1) - 1)));
     }
+
+    static class ParquetSchemaOptions
```

Comment:
`ParquetTester` is public, so this ( it's configuration parameter) should be public too